static-site/src/components/Footer/index.jsx

```diff
@@ -18,8 +18,8 @@ const SplashImagesCredit = () => (
         {` by Zhang `}<i>et al., </i>
         <a href="https://commons.wikimedia.org/wiki/Ebola#/media/File:Ebola_virus_em.png">Ebola EM</a>
         {` by Frederick Murphy / CDC, `}
-        <a href="https://en.wikipedia.org/wiki/File:Monkeypox.gif">monkeypox EM</a>
-        {` by University of Wisconsin-La Crosse, `}
+        <a href="https://web.archive.org/web/20081011105504/http://www.uwlax.edu/clinmicro/clinicalmicro.htm">mpox EM</a>
+        {` from unknown source `}
```

Comment:
@trvrb the current image has been [deleted from Wikimedia Commons](https://commons.wikimedia.org/wiki/Commons:Deletion_requests/File:Monkeypox.gif) since is from an unknown source. Would it be better to use [this properly attributed image](https://commons.wikimedia.org/wiki/File:Monkeypox_Virus_(52103767506).jpg) by applying the image styling as you did for d4fd7eec2c27b07dd276883cebfcf92dddfd3cf2?

Reply:
Sounds great. We should be being decently careful about not including copy written images here.

Reply:
I took a quick look and everything but mpox seem to be fine. I can add a comment in this file to be careful.

@trvrb for the new mpox image, can you detail what filters/effects you applied in LunaPic for the pathogen images, or upload directly to this PR?

Reply:
LunaPic filters etc [here](https://github.com/nextstrain/nextstrain.org/blob/master/static-site/README.md#adding-new-cards--tiles-for-the-splash-page)

Reply:
Oh perfect! I'll try it out.

Reply:
Done: a94e65bb595855798ba402647990c8c0b3198327